recipes-core/procd-base-files/procd-base-files_git.bb

```diff
@@ -0,0 +1,108 @@
+# this software, where such license applies only to those patent claims, already
+# acquired or hereafter acquired, licensable by such copyright holder or contributor
+# that are necessarily infringed by:
+#
+# (a) their Contribution(s) (the licensed copyrights of copyright holders and
+# non-copyrightable additions of contributors, in source or binary form) alone;
+# or
+#
+# (b) combination of their Contribution(s) with the work of authorship to which
+# such Contribution(s) was added by such copyright holder or contributor, if,
+# at the time the Contribution is added, such addition causes such combination
+# to be necessarily infringed. The patent license shall not apply to any other
+# combinations which include the Contribution.
+#
+# Except as expressly stated above, no rights or licenses from any copyright
+# holder or contributor is granted under this license, whether expressly, by
+# implication, estoppel or otherwise.
+#
+# DISCLAIMER
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDERS OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
+# USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+# NOTE
+#
+# This is part of a management middleware software package called MMX that was developed by Inango Systems Ltd.
+#
+# This version of MMX provides web and command-line management interfaces.
+#
+# Please contact us at Inango at support@inango-systems.com if you would like to hear more about
+# - other management packages, such as SNMP, TR-069 or Netconf
+# - how we can extend the data model to support all parts of your system
+# - professional sub-contract and customization services
+#
+################################################################################
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
+SUMMARY = "Scripts from procd (OpenWrt process management daemon)"
+DESCRIPTION = "Some scripts, i.e. rc.common, is needed to autostart OpenWrt services on RDKB"
+HOMEPAGE = "http://wiki.openwrt.org/doc/techref/procd"
+LICENSE = "BSD"
+LIC_FILES_CHKSUM = "file://procd.c;beginline=1;endline=13;md5=61e3657604f131a859b57a40f27a9d8e"
+SECTION = "base"
+DEPENDS = "libubox ubus json-c"
+
+SRC_URI = "git://git.openwrt.org/project/procd.git"
+SRCREV = "6acc48c7a2faac48c534b8a5516500c270550a9e"
+
+S = "${WORKDIR}/git"
+PD = "${S}/openwrt/package/system/procd/files"
```

Comment:
what is var PD and var BF ?

Reply:
That is subpathes to ProcD files (PD) and Base Files (BF) folders within `${S}`
Their goal is using short paths in `do_install` task

Reply:
Please resolve conversation if my explanation is enough

Reply:
Looks like I can't resolve conversations, please resolve by yourself